react/week1/todo-list/src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import Todos from './App';
+import reportWebVitals from './reportWebVitals';
+import getTodos from './getTodos'
+
+const todos = getTodos()
+
+function Render(){
+  return (
+    <div>
+      {todos.map((todo) => {
+        return <Todos description={todo.description} deadline={todo.deadline} />
```

Comment:
Here the console is showing a warning that says Each child in a list should have a unique "key" prop.
you can avoid the problem by providing a unique key that has the value of todo.id

Reply:
Thanks I will update my codes 👍